tests/micro/arduino/test_arduino_workflow.py

```diff
@@ -95,7 +95,7 @@ def test_model_platform_templating(project_dir, project):
         # TVM causes the amount of memory needed to decrease.
         workspace_size = int(workspace_size_defs[0])
         assert workspace_size < 30000
-        assert workspace_size > 10000
+        assert workspace_size > 9000
```

Comment:
cc @mehrdadh this gave an error in the upstream build
```[2023-02-16T17:18:08.641Z] E           assert 9776 > 10000```
Is it ok to reduce the minimum workspace size there?

Reply:
yeah that's fine. If it was the upper bound I would've been concerned.